jmix-flowui/flowui/src/main/resources/io/jmix/flowui/app/jmxconsole/mbean-attribute-detail-view.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+  ~ Copyright 2022 Haulmont.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<view xmlns="http://jmix.io/schema/flowui/view"
+      title="msg://mbeanAttributeDetailView.title">
+    <data>
+        <instance id="attrDc" class="io.jmix.flowui.app.jmxconsole.model.ManagedBeanAttribute"/>
+    </data>
+    <facets>
+        <dataLoadCoordinator auto="true"/>
+    </facets>
+    <actions>
+        <action id="saveCloseAction" type="detail_saveClose"/>
+        <action id="closeAction" type="detail_close"/>
+    </actions>
+    <layout>
+        <div id="valueContainerDiv"/>
```

Comment:
do we need a container? can't we add a component by index to the first place in a view?

P.S. `valueContainerDiv` is a bad id name. it'a a `fieldWrapper` or `valueComponentWrapper`